deploy/charts/operator/values.yaml

```diff
@@ -85,10 +86,10 @@ operator:
   resources:
     limits:
       cpu: 500m
-      memory: 128Mi
```

Comment:
Is there a reason we want to bump these up? You could override these values at deploy time if needed.